digital-credentials/allow-attribute.https.html

```diff
@@ -14,65 +16,149 @@
             const hostInfo = get_host_info();
             const iframeDetails = [
                 {
+                    action: "get",
                     policy: null,
                     crossOrigin: false,
                     expectIsAllowed: true,
                 },
                 {
+                    action: "get",
                     policy: null,
                     crossOrigin: true,
                     expectIsAllowed: false,
                 },
                 {
+                    action: "get",
                     policy: "digital-credentials-get",
                     crossOrigin: false,
                     expectIsAllowed: true,
                 },
                 {
+                    action: "get",
                     policy: "digital-credentials-get",
                     crossOrigin: true,
                     expectIsAllowed: true,
                 },
                 {
+                    action: "get",
                     policy: "digital-credentials-get *",
                     crossOrigin: false,
                     expectIsAllowed: true,
                 },
                 {
+                    action: "get",
                     policy: "digital-credentials-get *",
                     crossOrigin: true,
                     expectIsAllowed: true,
                 },
                 {
+                    action: "get",
                     policy: "digital-credentials-get 'none'",
                     crossOrigin: false,
                     expectIsAllowed: false,
                 },
                 {
+                    action: "get",
                     policy: "digital-credentials-get 'none'",
                     crossOrigin: true,
                     expectIsAllowed: false,
                 },
                 {
+                    action: "get",
                     policy: "digital-credentials-get 'self'",
                     crossOrigin: false,
                     expectIsAllowed: true,
                 },
                 {
+                    action: "get",
                     policy: "digital-credentials-get 'self'",
                     crossOrigin: true,
                     expectIsAllowed: false,
                 },
                 {
+                    action: "get",
                     policy: `digital-credentials-get ${hostInfo.HTTPS_REMOTE_ORIGIN}`,
                     crossOrigin: false,
                     expectIsAllowed: false,
                 },
                 {
+                    action: "get",
                     policy: `digital-credentials-get ${hostInfo.HTTPS_REMOTE_ORIGIN}`,
                     crossOrigin: true,
                     expectIsAllowed: true,
                 },
+                {
```

Comment:
Oh, I feel we should split this into two... I like the addition of action. That's a good idea.